go.sum

```diff
@@ -688,6 +688,16 @@ github.com/mandelsoft/goutils v0.0.0-20240605092610-3d3ecdc682ea h1:VDau62ZodJev
 github.com/mandelsoft/goutils v0.0.0-20240605092610-3d3ecdc682ea/go.mod h1:EbNqk9JceSMq7MJuALB/vlOpoD4MAGE0TenM9TR+C0o=
 github.com/mandelsoft/goutils v0.0.0-20240605100619-84d61423f421 h1:AdG9KVK4wloCoxotPRmaPgI01P+4N9tWpIT1W0QwsJU=
 github.com/mandelsoft/goutils v0.0.0-20240605100619-84d61423f421/go.mod h1:EbNqk9JceSMq7MJuALB/vlOpoD4MAGE0TenM9TR+C0o=
+github.com/mandelsoft/goutils v0.0.0-20240607120900-c70e5479360b h1:qtJDWErrFlctzuord4yZq8u2EYD+ReXqAfykM5UDhSU=
+github.com/mandelsoft/goutils v0.0.0-20240607120900-c70e5479360b/go.mod h1:9TJgkwSY43RWHiIAAz7fL8SEIHf0L13Pk4w8fDIt+i4=
+github.com/mandelsoft/goutils v0.0.0-20240608121958-b990a46c6aa2 h1:EC3lToqJdkAmtbkrizqVxAPMgbXEoJqppkdKF53aIAg=
+github.com/mandelsoft/goutils v0.0.0-20240608121958-b990a46c6aa2/go.mod h1:9TJgkwSY43RWHiIAAz7fL8SEIHf0L13Pk4w8fDIt+i4=
+github.com/mandelsoft/goutils v0.0.0-20240608130105-a79e05d3ba99 h1:DIQZb85AkMjIA3dTzLnW0LzxYSMryPq+n0mOJSLai/s=
+github.com/mandelsoft/goutils v0.0.0-20240608130105-a79e05d3ba99/go.mod h1:9TJgkwSY43RWHiIAAz7fL8SEIHf0L13Pk4w8fDIt+i4=
+github.com/mandelsoft/goutils v0.0.0-20240608130415-ae767b734156 h1:LLvRlPQgpzGtWbMvcvFmWBdzgNOn/9kBmiDu24kGncw=
+github.com/mandelsoft/goutils v0.0.0-20240608130415-ae767b734156/go.mod h1:9TJgkwSY43RWHiIAAz7fL8SEIHf0L13Pk4w8fDIt+i4=
+github.com/mandelsoft/goutils v0.0.0-20240608132424-ec9fb7fa611a h1:7ZnEdaPeoshk2I1GfZcE51NkSjGwfuVoM2Jf/tAXB2w=
+github.com/mandelsoft/goutils v0.0.0-20240608132424-ec9fb7fa611a/go.mod h1:9TJgkwSY43RWHiIAAz7fL8SEIHf0L13Pk4w8fDIt+i4=
```

Comment:
how does this `go.sum` work? Sometimes there are two entries per dependency, sometimes more than 2...